### PR TITLE
Auto focus added outputs to the commands

### PR DESCRIPTION
**Pull request by moishce**
## Status
- [x] In Progress
- [ ] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
fixes: https://github.com/demisto/etc/issues/33792

## Description
Auto focus added outputs to the commands

## Screenshots
Paste here any images that will help the reviewer

## Minimum version of Demisto
- [ ] 5.0.0
- [ ] 5.5.0
- [ ] 6.0.0

## Does it break backward compatibility?
   - [ ] Yes
       - Further details:
   - [ ] No

## Must have
- [ ] Tests
- [ ] Documentation

### CODE REVIEW

File: Packs/AutoFocus/Integrations/AutofocusV2/AutofocusV2.py [hunk 1]
```diff
@@ -1551,6 +1557,22 @@ def get_tags_for_generic_context(tags: Optional[list]):
     return results
 
 
+def get_tags_for_tags_and_malware_family_fields(tags: Optional[list], is_malware_family=False):
+    """get specific tags for the tgas and malware_family fields"""
+    if not tags:
+        return None
```
Comment (ShahafBenYakir) [inline Packs/AutoFocus/Integrations/AutofocusV2/AutofocusV2.py:1563]: Why `None` and not `[]`?

File: Packs/AutoFocus/Integrations/AutofocusV2/AutofocusV2.py [hunk 2]
```diff
@@ -1551,6 +1557,22 @@ def get_tags_for_generic_context(tags: Optional[list]):
     return results
 
 
+def get_tags_for_tags_and_malware_family_fields(tags: Optional[list], is_malware_family=False):
+    """get specific tags for the tgas and malware_family fields"""
```
Comment (ShahafBenYakir) [inline Packs/AutoFocus/Integrations/AutofocusV2/AutofocusV2.py:1561]: Add type strings

File: Packs/AutoFocus/Integrations/AutofocusV2/AutofocusV2.py [hunk 3]
```diff
@@ -1551,6 +1557,22 @@ def get_tags_for_generic_context(tags: Optional[list]):
...
+    results = []
+    for item in tags:
+        results.append(item.get('tag_name'))
+        results.append(item.get('public_tag_name'))
+        for alias in item.get('aliases', []):
+            results.append(alias)
+        if not is_malware_family:
+            for group in item.get('tagGroups', [{}]):
+                results.append(group.get('tag_group_name'))
+    return list(dict.fromkeys(remove_empty_elements(results)))
```
Comment (ShahafBenYakir) [inline Packs/AutoFocus/Integrations/AutofocusV2/AutofocusV2.py:1573]:
```suggestion
    return list(filter(None,results))
```

File: Packs/AutoFocus/Integrations/AutofocusV2/AutofocusV2.py [hunk 4]
```diff
@@ -1337,7 +1337,9 @@ def search_ip_command(ip):
 
         ip = Common.IP(
             ip=ip_address,
-            dbot_score=dbot_score
+            dbot_score=dbot_score,
+            malware_family=get_tags_for_tags_and_malware_family_fields(raw_tags, True),
+            tags=get_tags_for_tags_and_malware_family_fields(raw_tags)
```
Comment (ShahafBenYakir) [inline Packs/AutoFocus/Integrations/AutofocusV2/AutofocusV2.py:1342]: We are putting similar values here (and in all other places)?

Comment (moishce) [reply Packs/AutoFocus/Integrations/AutofocusV2/AutofocusV2.py:1342]: Yes, that's the request

File: Packs/AutoFocus/Integrations/AutofocusV2/AutofocusV2.py (hunk 3 above)
Comment (moishce) [reply Packs/AutoFocus/Integrations/AutofocusV2/AutofocusV2.py:1573]: I also need to delete duplicates

Comment (ShahafBenYakir) [reply Packs/AutoFocus/Integrations/AutofocusV2/AutofocusV2.py:1573]: So let's do this 
```
list(set(filter(None,results)))
```

File: Packs/AutoFocus/Integrations/AutofocusV2/AutofocusV2.py (hunk 1 above)
Comment (moishce) [reply Packs/AutoFocus/Integrations/AutofocusV2/AutofocusV2.py:1563]: In any case in common classes it checks if it is None then it does not output  the field to the context data

**Review by ShahafBenYakir — COMMENTED** [review]
Can we test the new outputs in the test PB?

**Review by ShahafBenYakir — APPROVED** [review]

**Pull request merged**